docs/supported-languages.md

```diff
@@ -295,6 +295,13 @@ Semgrep Supply Chain parses **lockfiles** for dependencies, then scans your code
    <td>Maven-generated dependency tree (See <a href="/docs/semgrep-supply-chain/setup-maven/">Setting up SSC scans for Apache Maven</a> for instructions.)</td>
    <td style={{"text-align": "center"}}>Lockfile-only</td>
    <td>--</td>
+</tr>
+<tr>
+   <td>Swift</td>
+   <td>SwiftPM</td>
+   <td>Swift-generated <code>Package.resolved</code> file. (See <a href="https://developer.apple.com/documentation/packagedescription/package/dependency">Apple Swift documentation </a> for instructions.)</td>
```

Comment:
I think this link would be more helpful https://www.swift.org/documentation/package-manager/

Reply:
I'll add it in!